release/release_logs/1.13.0rc0/benchmarks/many_actors.json

```diff
@@ -0,0 +1,15 @@
+{
+    "_peak_memory": 4.94,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n182\t1.54GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n1442\t0.85GiB\tpython distributed/test_many_actors.py\n207\t0.17GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n16\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n400\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n283\t0.1GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n1487\t0.06GiB\tray::MemoryMonitorActor.run()\n17\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=30e1d4bf\n200\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.74.7:9031 --host=0.0.0.0 -\n240\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m",
+    "actors_per_second": 298.0171917593177,
+    "num_actors": 10000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "actors_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 298.0171917593177
```

Comment:
@jjyao @scv119 IT was used to be 388, so there was about 30% regression. Was there any perf alert before?

Reply:
The latest run in master is `467.168` (https://buildkite.com/ray-project/release-tests-branch/builds/476#302bbe89-8f43-4560-bca7-4a053b1a6f4d). Based on the history, I feel this is false alarm.